node/src/blockchain/blockchain_bridge.rs

```diff
@@ -1211,17 +1488,16 @@ mod tests {
             amount: 4565,
             process_error: None,
         };
-        let blockchain_interface_mock = BlockchainInterfaceMock::default()
-            .get_transaction_receipt_params(&get_transaction_receipt_params_arc)
-            .get_transaction_receipt_result(Err(BlockchainError::QueryFailed("booga".to_string())));
+        let port = find_free_port();
+        let _blockchain_client_server = MBCSBuilder::new(port).start();
```

Comment:
You can add a comment as you did some other tests of this type, explaining that the lack of response from the server will cause the error. I quite like this approach.

Reply:
Would be nice to have this